### PR TITLE
[DOC release] Ember.HTMLBars.unbound

### LINKED ISSUE #11989: Document `unbound`
Changes to the behavior of `unbound` were made in: https://github.com/emberjs/ember.js/pull/11965

This helper has no docs, but we should add some now that it is roughly sane.

### PR DESCRIPTION
**Pull request by duggiefresh**
Closes #11989
[ci skip]

### CODE REVIEW

**Comment by mixonic** [comment]
Thanks! This should be on the `Ember.Templates.helper` class, like action for example:

* http://emberjs.com/api/classes/Ember.Templates.helpers.html#method_action
* https://github.com/emberjs/ember.js/blob/master/packages/ember-routing-htmlbars/lib/keywords/action.js

**Comment by duggiefresh** [comment]
Great, thanks for the feedback! I just updated the PR. Hopefully, this is what you were looking for. :)

File: packages/ember-htmlbars/lib/keywords/unbound.js [hunk 1]
```diff
@@ -1,8 +1,22 @@
 import Ember from 'ember-metal/core'; // assert
 
 /**
-@module ember
-@submodule ember-htmlbars
+  The `{{unbound}}` helper can be used with bound helper invocations to
+  render them in their unbound form.
+
+  ```handlebars
+  {{unbound capitalize name}}
```
Comment (mixonic) [inline packages/ember-htmlbars/lib/keywords/unbound.js]: Please use the format `{{unbound (capitalize name)}}`. This is the only supported syntax in 2.0.

**Comment by mixonic** [comment]
@duggiefresh seems good organization-wise, so long as you smoke tested.

**Comment by duggiefresh** [comment]
Just updated with the new syntax. Thanks again, @mixonic

File: packages/ember-htmlbars/lib/keywords/unbound.js [hunk 2]
```diff
@@ -1,8 +1,22 @@
...
+
+  ```handlebars
+  {{unbound (capitalize name)}}
+  ```
+
+  In the aforementioned example, if the `name` property changes, the helper
+  will not re-render.
+
+  @module ember
+  @submodule ember-htmlbars
```
Comment (mixonic) [inline packages/ember-htmlbars/lib/keywords/unbound.js]: As in https://github.com/emberjs/ember.js/blob/master/packages/ember-routing-htmlbars/lib/keywords/action.js#L3 the submodule should be `ember-templates`

Comment (duggiefresh) [inline packages/ember-htmlbars/lib/keywords/unbound.js]: Ha. Sorry, I'm having one of those Fridays.

**Comment by mixonic** [comment]
Gracias :-)

**Comment by mixonic** [comment]
@duggiefresh hey I'm curious, what is your use-case for `unbound`? I just did a bunch of work on it, and we would like to remove it in the future (I guess 3.0).

**Comment by duggiefresh** [comment]
@mixonic No strong use case. TBH I submitted the PR because it was an open issue.

**Comment by mixonic** [comment]
:+1: thanks man :-)

**Pull request merged**